torchmultimodal/models/flava.py

```diff
@@ -29,11 +34,7 @@
 
 
 EMBEDDING_OPTIONS = Literal["image", "text", "mm"]
-TransformerOutput = namedtuple(
-    "TransformerOutput",
-    ["last_hidden_state", "pooler_output", "hidden_states", "attentions"],
-    defaults=(None, None, None, None),
-)
+
 FLAVAOutput = namedtuple(
     "FLAVAOutput",
     ["image", "image_masked", "text", "text_masked", "multimodal", "multimodal_masked"],
```

Comment:
I'm thinking a cleaner way to define a named tuple is to do it just like `TwoTowerOutput` [here](https://github.com/facebookresearch/multimodal/blob/f03c241905ed276be4db0f40ce3fc535118ddd2c/torchmultimodal/architectures/two_tower.py#L14). That way it's consistent across the codebase and you don't have to define the type annotations separately. What are your thoughts? @ankitade

Reply:
yeah we can refactor this eventually. I am tempted to just do dataclasses though. The two tower one uses namedtuple because dataclasses are not torchscript friendly but thats not restriction for flava

Reply:
ok I will leave this for another PR